### PR TITLE
Fix local image building in tests

### PR DESCRIPTION
**Pull request by Rozzii**

### CODE REVIEW

File: jenkins/scripts/integration_test_env.sh [hunk 1]
```diff
@@ -13,26 +13,24 @@ then
   # for CAPM3 to verify the process (not BMO due to the build time for BMO image)
 
   if [[ "${UPDATED_BRANCH}" == "${REPO_BRANCH}" ]] && [[ "${UPDATED_REPO}" == *"${REPO_ORG}/${REPO_NAME}"* ]]; then
-    export BAREMETAL_OPERATOR_LOCAL_IMAGE="https://github.com/metal3-io/baremetal-operator.git"
-    export CAPM3_LOCAL_IMAGE="https://github.com/metal3-io/cluster-api-provider-metal3.git"
-    export CAPM3_LOCAL_IMAGE_BRANCH="${CAPM3RELEASEBRANCH}"
-    export BMOBRANCH="main"
```
Comment (lentzi90) [inline jenkins/scripts/integration_test_env.sh:19]: I think we should still set `BMOBRANCH` here. From what I could find in metal3-dev-env, the `BMOBRANCH` is by default set to `BMORELEASE`. See https://github.com/metal3-io/metal3-dev-env/blob/5c29964fedbe3e9c8b47b16d537495787df6c409/lib/releases.sh#L49

Comment (Rozzii) [reply jenkins/scripts/integration_test_env.sh:19]: yep you are right

Comment (lentzi90) [reply jenkins/scripts/integration_test_env.sh:19]: I don't understand. Why would there not be a release?

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Review by kashifest — APPROVED** [review]

**Pull request merged**